SmartEVSE-3/data/update2.html

```diff
@@ -75,7 +75,7 @@
             <tr>
               <td>Factory</td>
               <td>stable</td>
-              <td><a href="https://github.com/"
+              <td><a href="https://github.com/SmartEVSE/SmartEVSE-3"
```

Comment:
I think this will cause a problem; this reference (called this.href) will have  "OWNER_FACT + / + REPO _FACT added to it, so the link would become:
https://github.com/SmartEVSE/SmartEVSE-3/SmartEVSE/SmartEVSE-3   ....

Reply:
Well, if that was the case this would not need a fix 🙂 
I came across this in the update page. Both links just brought me to GitHub.com

Reply:
Well thats strange, in all of my versions, if you hover your mouse over the link it SHOWS https://github.com but if you click it it actually brings you to the right repository (dingo35/SmartEVSE-3.5 or SmartEVSE/SmartEVSE-3.5).

Does yours behave differently? If so, what browser do you use?

Reply:
Seems you are correct. I was assuming the hovering url was also where it would take me.

I will adjust the PR accordingly!